Autocoders/Python/src/fprime_ac/generators/templates/enums/enum_cpp.tmpl

```diff
@@ -52,6 +52,34 @@ namespace ${namespace} {
     return *this;
   }
 
+  ${name}& ${name} :: operator=(const t a)
+  {
+#for $item_name,$item_value,$item_comment in $items_list:
```

Comment:
This loop logic doesn't look right. It re-assigns true and false, depending on whether a value is specified. For example `enum { A =1 , B, C = 1 }` would assign true, false, true. The net effect is to assign the result based on the last value. So why assign over and over?

Reply:
I suspect the intent is to assign manual = true if there is any manual value, but this code doesn't do that. Was this code copy-pasted from somewhere, and does that code have the same issue?